### PR TITLE
Views

### PR DESCRIPTION
**Pull request by cmolinan**
####  Project requirements done
- Implementation of the design from the sneak peek wireframes. Focus on functionality
- Use of methods created before

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,15 @@
...
+  <p><%= @post.text %></p>  
+</div>
+
+<div class='div-comments'>
+  <% @post.five_recent_comments(@post.id).each do |comment| %>
+     <div class='comments'>
+        <span style="font-weight :bold font-size: 10px">Username <%= comment.user_id %>:</span> <%= comment.text %>
+      </div>
+  <% end %>        
+</div>
```
Comment (danmainah) [inline app/views/posts/show.html.erb]: - [ ] @cmolinan great work on your views, however, the design in your post-show page doesn't match the wireframe given, below is a comparison

Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
image  |  image

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+
+    <div class='div-comments'>
+    <% post.five_recent_comments(post.id).each_with_index do |comment, index| %>
+       <div class='comments'>
+          <span style="font-weight :bold font-size: 10px">Username <%= comment.user_id %>:</span> <%= comment.text %>
+        </div>
+    <% end %>        
+    </div>
+          
+<% end %>
```
Comment (danmainah) [inline app/views/posts/index.html.erb:20]: - [ ] Kindly ensure the design of your posts page matches the wireframe given, and check the positioning of comments and likes:

Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @cmolinan , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/views/posts/show.html.erb (hunk 1 above)
Comment (cmolinan) [reply app/views/posts/show.html.erb]: @danmainah Done !

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (cmolinan) [reply app/views/posts/index.html.erb:20]: @danmainah Done !

**Review by omarsalem7 — APPROVED** [review]
Hi @cmolinan 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by cmolinan** [comment]
@omarsalem7 Thank you for yor review !

**Pull request merged**